modules/gapi/src/backends/fluid/gfluidcore_simd_sse42.hpp

```diff
@@ -0,0 +1,547 @@
+        // pixel_3
+        vec.val = _mm_insert_epi32(vec.val, *reinterpret_cast<const int*>(&src[chanNum * (*(index + 2) + pos)]), 2);
+        // pixel_4
+        vec.val = _mm_insert_epi32(vec.val, *reinterpret_cast<const int*>(&src[chanNum * (*(index + 3) + pos)]), 3);
+    }
+}  // namespace
+
+CV_ALWAYS_INLINE void v_set_alpha(const short* alpha, v_int16x8& a1, v_int16x8& a2)
+{
+    a1.val = _mm_setr_epi16(*alpha, *alpha, *alpha, *(alpha + 1), *(alpha + 1), *(alpha + 1),
+                            *(alpha + 2), *(alpha + 2));
+    a2.val = _mm_setr_epi16(*(alpha + 2), *(alpha + 3), *(alpha + 3), *(alpha + 3),
+                            *(alpha + 4), *(alpha + 4), *(alpha + 4), 0);
+}
+
+CV_ALWAYS_INLINE void vertical_anyLPI(const uchar* src0, const uchar* src1,
+                                      uchar* tmp, const int inLength,
+                                      const short beta) {
+    constexpr int nlanes = static_cast<int>(v_uint8::nlanes);
+
+    const int half_nlanes = nlanes/2;
+    int w = 0;
+    for (;;) {
+        for (; w <= inLength - nlanes; w += nlanes) {
+            v_int16 s0 = v_reinterpret_as_s16(vx_load_expand(&src0[w]));
+            v_int16 s1 = v_reinterpret_as_s16(vx_load_expand(&src1[w]));
+            v_int16 s2 = v_reinterpret_as_s16(vx_load_expand(&src0[w + half_nlanes]));
+            v_int16 s3 = v_reinterpret_as_s16(vx_load_expand(&src1[w + half_nlanes]));
+            v_int16 res1 = v_mulhrs(s0 - s1, beta) + s1;
+            v_int16 res2 = v_mulhrs(s2 - s3, beta) + s3;
+
+            vx_store(tmp + w, v_pack_u(res1, res2));
+        }
+
+        if (w < inLength) {
+            w = inLength - nlanes;
+            continue;
+        }
+        break;
+    }
+}
+
+template<int chanNum>
+CV_ALWAYS_INLINE void horizontal_anyLPI(uint8_t* dst,
+                                        const uchar* src, const short mapsx[],
+                                        const short alpha[], const int width)
+{
+    constexpr int nlanes = static_cast<int>(v_uint8::nlanes);
+    constexpr int pixels = nlanes / 3;
+
+    int x = 0;
+    v_uint16 a1, a2, b1, b2;
+    v_uint8 a, b;
+    v_int16 a00, a01;
+    for (; width >= pixels;) {
+        for (; x <= width - pixels; x += pixels) {
+            v_set_alpha(&alpha[x], a00, a01);
+            v_gather_pixel_map<chanNum>(a, src, &mapsx[x], 0);
+            v_gather_pixel_map<chanNum>(b, src, &mapsx[x], 1);
+
+            v_expand(a, a1, a2);
+            v_expand(b, b1, b2);
+            v_int16 a_1 = v_reinterpret_as_s16(a1);
+            v_int16 a_2 = v_reinterpret_as_s16(a2);
+            v_int16 b_1 = v_reinterpret_as_s16(b1);
+            v_int16 b_2 = v_reinterpret_as_s16(b2);
+
+            v_int16 r_1 = v_mulhrs(a_1 - b_1, a00) + b_1;
+            v_int16 r_2 = v_mulhrs(a_2 - b_2, a01) + b_2;
+
+            v_uint8 res = v_pack_u(r_1, r_2);
+            vx_store(&dst[chanNum * x], res);
```

Comment:
Looks like it is crashed here (pixels = 15, but sizeof(res) = 16 - write after buffer):

<details>

```
[ RUN      ] ResizeTestFxFyFluid/ResizeTestFxFy.AccuracyTest/16, where GetParam() = (8UC3, 640x480, SAME_TYPE, 0x100264700, Tolerance_FloatRel_IntAbs(1e-05, 1), 1, 2, 1)
2021-10-01 16:33:36.262161+0000 opencv_test_gapi[36769:40729401] dynamic_cast error 2: One or more of the following type_info's has hidden visibility or is defined in more than one translation unit. They should all have public visibility. N2cv4util3any6holderE, N2cv4util3any11holder_implINS_4gapi8use_onlyEEE, N2cv4util3any11holder_implINS_4gapi8use_onlyEEE.
Process 36769 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x109ac2000)
    frame #0: 0x0000000101336056 libopencv_gapi.4.5.dylib`cv::gapi::fluid::GFluidResize::run(cv::gapi::fluid::View const&, cv::Size_<int>, double, double, int, cv::gapi::fluid::Buffer&, cv::gapi::fluid::Buffer&) + 3462
libopencv_gapi.4.5.dylib`cv::gapi::fluid::GFluidResize::run:
->  0x101336056 <+3462>: movdqu %xmm5, (%r12)
    0x10133605c <+3468>: addq   $0x5, %r9
    0x101336060 <+3472>: addq   $0xf, %r12
    0x101336064 <+3476>: cmpq   %r14, %r9
Target 0: (opencv_test_gapi) stopped.
(lldb) register read
General Purpose Registers:
       rax = 0x0000000106de0880
       rbx = 0x000000000000077a
       rcx = 0x0000000000000777
       rdx = 0x0000000000000777
       rdi = 0x000000000000077a
       rsi = 0x000000000000077a
       rbp = 0x00007ffeefbfed70
       rsp = 0x00007ffeefbfec50
        r8 = 0x000000000000027e
        r9 = 0x00000000000004fb
       r10 = 0x0000000109ac1100
       r11 = 0x00000000000004fb
       r12 = 0x0000000109ac1ff1
       r13 = 0x0000000105030400
       r14 = 0x00000000000004fb
       r15 = 0x0000000105033608
       rip = 0x0000000101336056  libopencv_gapi.4.5.dylib`cv::gapi::fluid::GFluidResize::run(cv::gapi::fluid::View const&, cv::Size_<int>, double, double, int, cv::gapi::fluid::Buffer&, cv::gapi::fluid::Buffer&) + 3462
    rflags = 0x0000000000010246
        cs = 0x000000000000002b
        fs = 0x0000000000000000
        gs = 0x0000000000000000
```

</details>

Reply:
Ok. Fixed.